src/test/java/com/tvtracker/enterprise/EnterpriseApplicationTests.java

```diff
@@ -20,28 +20,29 @@
 
     final String TEST_USERNAME = "testUser";
     final String TEST_USER_PASSWORD = "testPassword";
+    MediaEntry mediaEntry = new MediaEntry();
+    UserAccount userAccount = new UserAccount();
 
     @Test
     void contextLoads() {
     }
 
     @Test
     void userCreatesUserAccount_ReturnsValidAuthenticationToken() throws Exception {
-        UserAccount userAccount = whenUserSendsUserAccountWithUniqueUsername();
+        userAccount = whenUserSendsUserAccountWithUniqueUsername();
```

Comment:
No need to declare the object anymore as its been declared above at class level.